soliket/halo_model/halo_model.py

```diff
@@ -0,0 +1,174 @@
+"""
+.. module:: soliket.halo_model
+
+:Synopsis: Class to calculate Halo Models for non-linear power spectra.
+:Author: Ian Harrison
+
+.. |br| raw:: html
+
+   <br />
+
+
+Usage
+-----
+
+Halo Models for calculating non-linear power spectra for use in large scale structure 
+and lensing likelihoods. The abstract HaloModel base class should be built on with 
+specific model implementations. HaloModels can be added as theory codes alongside others 
+in your run settings. e.g.:
+
+.. code-block:: yaml
+
+  theory:
+    camb:
+    soliket.halo_model.HaloModel_pyhm:
+
+
+Implementing your own bias model
```

Comment:
is this really bias model or just a typo?

Reply:
Oops, just leftover from copy+paste.